.github/workflows/npm-publish-beta.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and Publish NPM Package
+
+on:
+    push:
+        branches:
+            - develop
```

Comment:
Just to confirm, @JohnnyTMD  won't you have to include the master branch in the list of branches here?

Reply:
hmm, maybe
the thing is that I had no way to test that previously

Reply:
I will merge this one and if the pipeline will not be triggered I will add a hotfix

Reply:
Alright, that sounds good too

Reply:
I have added the fixes now,